src/Interfaces/AbstractPolytope.jl

```diff
@@ -96,6 +97,60 @@ function isempty(P::AbstractPolytope)::Bool
     return isempty(vertices_list(P))
 end
 
+"""
+    chebyshev_center(P::AbstractPolytope{N};
+                     backend=default_polyhedra_backend(P, N),
+                     compute_radius::Bool=false) where {N<:Real}
+
+Compute the [Chebyshev center](https://en.wikipedia.org/wiki/Chebyshev_center)
+of a polytope.
+
+### Input
+
+- `P`              -- polytope
+- `backend`        -- (optional, default: `default_polyhedra_backend(P, N)`) the
+                      backend for polyhedral computations
+- `compute_radius` -- (optional; default: `false`) option to additionally return
+                      the radius of the largest ball enclosed by `P` around the
+                      Chebyshev center
+
+### Output
+
+If `compute_radius` is `false`, the result is the Chebyshev center of `P`.
+If `compute_radius` is `true`, the result is the pair `(c, r)` where `c` is the
+Chebyshev center of `P` and `r` is the radius of the largest ball with center
+`c` enclosed by `P`.
+
+### Notes
+
+The Chebyshev center is the center of a largest Euclidean ball enclosed by `P`.
+In general, the center of such a ball is not unique (but the radius is).
+
+Formally, letting ``∂P`` denote the boundary of ``P``, the Chebyshev center is
+
+```math
+    \\arg\\min_c \\min_{x ∈ ∂P} ‖x - c‖₂.
+```
+"""
+function chebyshev_center(P::AbstractPolytope{N};
+                          backend=default_polyhedra_backend(P, N),
+                          compute_radius::Bool=false
+                         ) where {N<:Real}
+    require(:Polyhedra; fun_name="chebyshev_center")
+    require(:JuMP; fun_name="chebyshev_center")
+    require(:GLPK; fun_name="chebyshev_center")
```

Comment:
lines 140 and 141 can be removed, as one can do:

```julia
julia> using LazySets, Polyhedra

julia> chebyshev_center(convert(HPolytope, rand(HPolygon)))
2-element Array{Float64,1}:
 -2.5065814803100483
  3.4125300460193526
```

Reply:
This is because [we load those packages with `Polyhedra`](https://github.com/JuliaReach/LazySets.jl/blob/bbda6f23830d5cd90efba23295a0a3ac9cd6a07c/src/Initialization/init_Polyhedra.jl#L6).